cleverhans/utils_tf.py

```diff
@@ -34,7 +34,13 @@ def model_loss(y, model, mean=True):
     else:
         logits = model
 
-    out = tf.nn.softmax_cross_entropy_with_logits(logits=logits, labels=y)
+    try:
+        y = tf.stop_gradient(y)
+        out = tf.nn.softmax_cross_entropy_with_logits_v2(
+                                                logits=logits, labels=y)
+    except AttributeError:
+        out = tf.nn.softmax_cross_entropy_with_logits(
```

Comment:
It would be great to add a warning here saying that this version of TF will not be supported by CleverHans in the future

Reply:
Sounds good, will do that.

Reply:
Please merge the PR, after i add the warning will submit a new PR for TF depreciation warnings.(there are warnings other than keep_dims, will resolve all in new PR )

Reply:
Thanks, I'll do that! Looks like your last commit broke the diff though (probably due to a conflict?)

Reply:
Yes there was some conflict, everything is sorted now.